extension/src/experiments/columns/like.ts

```diff
@@ -0,0 +1,22 @@
+import { Column } from '../webview/contract'
+
+export type ColumnLike = { label: string; path: string; types?: string[] }
+
+const starredColumnLike: ColumnLike = {
+  label: '$(star-full)',
+  path: 'starred',
+  types: ['boolean']
+}
+
+export const addStarredToColumns = (
+  columns: Column[] | undefined
+): ColumnLike[] | undefined => {
+  if (!columns?.length) {
```

Comment:
Didn't we have a lint rule where it said we have to compare length to 0?

Reply:
I think the optional chain will be messing with it. You would need to write 
`columns?.length === undefined || columns.length === 0`